v8/spnego/krb5Token.go

```diff
@@ -52,7 +52,10 @@ func (m *KRB5Token) Marshal() ([]byte, error) {
 			return []byte{}, fmt.Errorf("error marshalling AP_REQ for MechToken: %v", err)
 		}
 	case TOK_ID_KRB_AP_REP:
-		return []byte{}, errors.New("marshal of AP_REP GSSAPI MechToken not supported by gokrb5")
+		tb, err = m.APRep.Marshal()
+		if err != nil {
+			return []byte{}, fmt.Errorf("error marshalling AP_REQ for MechToken: %v", err)
```

Comment:
Consider using `%w` instead of `%v` for wrapping errors so the calling party can unwrap them if needed.

Reply:
I'm using `%v` to be consistent with upstream (`%v` is used in the other switch branches)